openedx/features/smart_referral/tasks.py

```diff
@@ -15,3 +17,14 @@ def task_send_referral_and_toolkit_emails(contact_emails, user_email):
         })
 
     mandrill_client.send_mail(MandrillClient.REFERRAL_SOCIAL_IMPACT_TOOLKIT, user_email, context={})
+
+
+@task(routing_key=settings.HIGH_PRIORITY_QUEUE)
+def task_send_referral_follow_up_email(contact_email_list):
```

Comment:
let's rename this function to `task_send_referral_follow_up_email**s**` like in its respective test. Because its sending multiple emails.

Reply:
done